lib/index.ts

```diff
@@ -70,7 +70,7 @@ export const useStoryblok = (
 export const useStoryblokState = <T = void>(
   initialStory: ISbStoryData<T> | null = null as ISbStoryData<T>,
   bridgeOptions: StoryblokBridgeConfigV2 = {}
-): ISbStoryData<T> => {
+): ISbStoryData<T> | undefined => {
```

Comment:
@fabb as `initialStory` is `ISbStoryData<T> | null`, shouldn't it be the same here?

Reply:
Thanks for the fix @fabb!
Can you explain the reason of the other two changes?

Also, we may need to change your PR, or implement the changes you suggest by ourselves, as we use semantic release for the SDK, so we need a specific format in the commit messages.
https://semantic-release.gitbook.io/semantic-release/#commit-message-format

Reply:
Thank you very much for the contribution @fabb!